features/admin/organisation_reports/without_users.feature

```diff
@@ -51,6 +52,22 @@ Feature: Orphans UI
     And I visit the invite users to become admin of organisations page
     Then I should see the preview email
 
+  Scenario: Super Admin can enter edit email template page
```

Comment:
great scenarios - perhaps move to another feature file?  Or possibly rename this feature file - "without_users" doesn't really tell us what it does ...